llvm/test/MC/EraVM/encoding/jump-table.ll

```diff
@@ -0,0 +1,61 @@
+; RUN: llc < %s | FileCheck %s
+; RUN: llc -filetype=obj -o %t.o < %s
+; RUN: llvm-readelf --sections --relocs --syms %t.o | FileCheck --check-prefix=ELF %s
+
+target datalayout = "E-p:256:256-i8:256:256:256-i256:256:256-S32-a:256:256"
```

Comment:
Please remove `i8` alignment. It must be natural. We've already addressed it in other tests.

Reply:
Fixed, thank you! I has the same issue in `data.ll` test - I plan sending a bunch of tiny adjustments to the tests anyway, so not changing unrelated test it in this PR.